assets/_scss/_navigation_additional-class.scss

```
@@ -86,6 +86,32 @@
 		}
 }
 
+/******************************************************
+ * Text inline & Vertical with hr
+ * 線の色制御
+ */
+
+:is( *[class*='nav--text-inline'],*[class*='nav--vertical-with-hr'] ) {
+
+	// とりあえず通常の線の色を定義
```

Stylelint (17.12.0)

[error] Unexpected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)